Documentation/Ctrl/Properties/Security.rst

```diff
@@ -21,8 +21,9 @@ security
     ..  code-block:: php
         :caption: EXT:core/Configuration/TCA/sys_file.php
```

Comment:
I would suggest to make an example on how to use this in my_extension in an override instead. This also removes the nessesarity of comenting out parts etc

Reply:
I will add an override example as addition. In custom extensions you have mostly no overrides of your own TCA, but instead integrate it into the default TCA of the table.

Reply:
Then I would still make an example with an own extension